system-int-test/test/cmr/system_int_test/search/facets/granule_v2_facets_search_test.clj

```diff
@@ -325,3 +325,111 @@
+        gran6 (ingest-granule-with-track "PROV2" coll2
+                                         {:cycle 42
+                                          :passes [{:pass 1 :tiles ["1L"]}]})
+        ;; granules for testing Full track tiles and multiple tiles
+        gran7 (ingest-granule-with-track "PROV2" coll2
+                                         {:cycle 3
+                                          :passes [{:pass 3 :tiles ["1L"]}
+                                                   {:pass 4 :tiles ["1L"]}]})
+        gran8 (ingest-granule-with-track "PROV2" coll2
+                                         {:cycle 3
+                                          :passes [{:pass 3 :tiles ["1R"]}
+                                                   {:pass 4 :tiles ["2L"]}]})
+        gran9 (ingest-granule-with-track "PROV2" coll2
+                                         {:cycle 3
+                                          :passes [{:pass 3 :tiles ["1F"]}
+                                                   {:pass 4 :tiles ["2L"]}]})
+
+        gran10 (ingest-granule-with-track "PROV2" coll2
+                                          {:cycle 3
+                                           :passes [{:pass 3 :tiles ["5F" "6R"]}
+                                                    {:pass 4 :tiles ["5L" "6R"]}]})
+        gran11 (ingest-granule-with-track "PROV2" coll2
+                                          {:cycle 42
+                                           :passes [{:pass 3 :tiles ["5R" "6L"]}
+                                                    {:pass 4 :tiles ["5L" "6R" "8F"]}]})]
+    (index/wait-until-indexed)
+
+    (let [root-node (search-and-return-v2-facets {:collection-concept-id coll2-concept-id
+                                                  :page_size 10})
+          cycle-node (-> root-node :children first)
+          cycle-facets (:children cycle-node)]
+      (testing "Facet structure correct"
+        (is (= "Browse Granules" (:title root-node)))
+        (is (= "group" (:type root-node)))
+        (is (= true (:has_children root-node)))
+        (is (= "Cycle" (:title cycle-node)))
+        (is (= "group" (:type cycle-node)))
+        (is (= true (:has_children cycle-node))))
+      (testing "Cycles returned in ascending order"
+        (is (= ["3" "42"] (map :title cycle-facets))))
+      (testing "Cycles have a type of filter"
+        (is (= ["filter" "filter"] (map :type cycle-facets))))
+      (testing "Counts correct"
+        (util/are3
+          [title cnt]
+          (is (= cnt (get-count-by-title cycle-facets title)))
+          "3" "3" 6
+          "42" "42" 2))
+      (testing "Traversals"
+        (let [cycle-selected-result (search-and-return-v2-facets
+                                      {:collection-concept-id coll2-concept-id
+                                       :page_size 10
+                                       :cycle 42})]
+          (is (= {:title "Browse Granules"
```

Comment:
We should have test to show that the pass links exist and we can traverse the pass link.